shaman/controllers/search.py

```diff
@@ -1,8 +1,22 @@
-from pecan import expose
+from pecan import expose, abort
+from shaman.models import Repo, Project
+from sqlalchemy import desc
 
 
 class SearchController(object):
 
+    def __init__(self):
+        self.filters = {
```

Comment:
I didn't add it here because it doesn't work with how we are applying the filters. We filter the `Repo` by the string value of these keys.

SInce `Project` is in itself another model, I wasn't able to find a way to just add it here and apply it in the same way. You have to retrieve it first and then apply it:

    project = Project.filter_by(name='ceph').first()

Which I do later in `apply_filters`